biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/embl/EmblId.java

```diff
@@ -0,0 +1,77 @@
+package org.biojava.nbio.core.sequence.io.embl;
+
+/**
+ * This class should parse the data of embl file identification
+ * @author Noor Aldeen Al Mbaidin
+ */
+public class EmblId {
+
+
+    private String PrimaryAccession;
```

Comment:
By convention, variable names in java should start with lower case.

Reply:
sorry for that.